Documentation/ApiOverview/Events/Events/Backend/ModifyEditFormUserAccessEvent.rst

```diff
@@ -0,0 +1,74 @@
+..  include:: /Includes.rst.txt
+..  index:: Events; ModifyEditFormUserAccessEvent
+..  _ModifyEditFormUserAccessEvent:
+
+=============================
+ModifyEditFormUserAccessEvent
+=============================
+
+..  versionadded:: 12.0
+    The PSR-14 event
+    :php:`TYPO3\CMS\Backend\Form\Event\ModifyEditFormUserAccessEvent\ModifyEditFormUserAccessEvent`
+    has been introduced which serves as a more powerful and flexible alternative
+    for the removed :php:`$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['typo3/alt_doc.php']['makeEditForm_accessCheck']`
+    hook.
+
+The event provides the full database row of the record in question next to the
+exception, which might have been set by the Core. Additionally, the event allows
+to modify the user access decision in an object-oriented way, using
+convenience methods.
+
+To modify the user access, the following methods are available:
+
+-   :php:`allowUserAccess()`: Allows user access to the editing form
```

Comment:
I usually handle it like this: Either the descriptions of methods are trivial, then they can be omitted or they are important, then they should be added as doc comments into the Event's code. This way they will be rendered into the API and also be available uppon code inspection and in IDE's. See how I handled this with https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/pull/2200 and https://review.typo3.org/c/Packages/TYPO3.CMS/+/75662